src/components/FeedCard.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import pintsLogo from '../assets/pintsLogo.svg';
+import quesLogo from '../assets/quesLogo.svg';
+import humanLogo from '../assets/humanLogo.svg';
+import clock from '../assets/clock.svg';
+import smallBlood from '../assets/smallBlood.svg';
+import smallTahaleb from '../assets/smallTahaleb.svg';
+
+const FeedCard = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleDivSizeChange = () => {
+    setOpen(!open);
+  };
+  const formatDate = (string) => {
+    let options = { year: 'numeric', month: 'short', day: 'numeric' };
+    return new Date(string).toLocaleDateString([], options);
+  };
+
+  const feed = [
+    {
+      fulfilled: true,
+      user: {
+        username: 'Jolie',
+        avatar:
+          'https://user-images.githubusercontent.com/12988551/108382684-1f03f000-721a-11eb-901b-3419f392e548.png',
+      },
+      data: {
+        operation: 'Request',
+        date: '2021-02-18T15:47:40.314Z',
+        reason: 'Car accident',
+        amount: '2 pints',
+        donors: ['Ahmed', 'Layla', 'jana', 'nabi8a'],
+      },
+    },
+  ];
+
+  return (
+    <div>
+      {feed.map((f) => (
+        <div
+          className={'container ' + (open ? 'expand' : '')}
```

Comment:
nice use of the expand function here. It could use a `transition` in CSS too to make it animate too. Or you can use tailwind classed 

https://tailwindcss.com/docs/transition-property

Also it seems this container class is made by you which overrides a very important tailwind class called container too. Check my comment below, and try to depend on tailwind only for styling. it can be done and I can help if there is no clue